sale_cancel_reason/wizard/cancel_reason_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_sale_order_cancel" model="ir.ui.view">
+        <field name="name">Reason for the cancellation</field>
+        <field name="model">sale.order.cancel</field>
+        <field name="inherit_id" ref="sale.sale_order_cancel_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//footer" position="before">
+                <br />
+                <p>
+                        Choose the reason for the cancellation of the
+                        sale order.
+                    </p>
+                <group>
+                    <field name="reason_id" widget="selection" />
```

Comment:
I prefer this approach instead of using the selection widget. Could you update it? This is better for the UI and allows users to search within the records.
```suggestion
                    <field name="reason_id" options="{'no_create': True, 'no_open': True}" />
```